### ISSUE TITLE
Share to policy across muti server instances? 

### ISSUE THREAD

**Issue by willswl** [mokies/ratelimitj#17]
I need to run a same service with multiple instances to run a rate limit cluster, for high-availabilty and high performance purpose, is that supported?

**Comment by mokies** [comment]
Yes. As long as the key is the same across instances and you are using the Redis based rate limiter than the library will support HA.
